aiida_quantumespresso/parsers/pp.py

```diff
@@ -102,9 +112,12 @@ def parse(self, **kwargs):
 
         # The following check should in principle always succeed since the iflag should in principle be set by the
         # `PpCalculation` plugin which only ever sets 0 - 4, but we check in order for the code not to except.
-        iflag = self.node.inputs.parameters.get_attribute('PLOT')['iflag']
-        if iflag not in range(5):
-            return self.exit_codes.ERROR_UNSUPPORTED_DATAFILE_FORMAT
+        # NOTE: The previous is no longer valid, needs to be corrected
+        plot_namelist = self.node.inputs.parameters.get_attribute('PLOT', None)
+        if plot_namelist is not None:
+            iflag = plot_namelist['iflag']
+            if iflag not in range(5):
+                return self.exit_codes.ERROR_UNSUPPORTED_DATAFILE_FORMAT
```

Comment:
Now the `PLOT` namelist might be missing, but if it is there then it should have the complying `iflag`.